src/argparse.cpp

```diff
@@ -2,15 +2,21 @@
 #include <iostream>
 #include <fstream>
 #include "cap32.h"
+#include "argparse.h"
 #include "stringutils.h"
 
 const struct option long_options[] =
 {
-   {"version", no_argument, nullptr, 'V'},
-   {"help",    no_argument, nullptr, 'h'},
-   {nullptr, 0,  nullptr, 0},
+   {"cfg_file", required_argument, nullptr, 'c'},
+   {"version",  no_argument, nullptr, 'V'},
+   {"help",     no_argument, nullptr, 'h'},
+   {nullptr, 0, nullptr, 0},
 };
 
+void initArgs(struct capriceArgs& args)
```

Comment:
At least I'll learn quite a bit about C++ when playing around with Caprice32 :-)
The proper way as you mention would be to make capriceArgs a class with a constructor. Some other possible common arguments (e.g. --fullscreen) would be boolean and need initialization. Other might be strings that we want to initialize to some non-empty values. I'll find some time to create the class in the next days and will update this pull request with it.